YR/PAN-OS-Policy-Optimizer-V10.2.0/XSUP-25940/CIAC-7392 (#28408)
demisto/content: Packs/PANOSPolicyOptimizer/ReleaseNotes/1_1_9.md

#### Integrations

##### PAN-OS Policy Optimizer (Beta)

> - You can now use the PAN-OS Policy Optimizer (Beta) integration with version 10.2.0 of PAN-OS.

Reviewer: ```suggestion
- Added support for version 10.2.0 of PAN-OS.```
